docs/products/storage/object-storage/guides/aws-sdk-for-php/index.md

```diff
@@ -0,0 +1,210 @@
+
+    $client->createBucket(['Bucket' => 'example-bucket']);
+
+## Delete a Bucket
+
+Deletes the specified *empty* bucket. If the bucket still contains objects, they must be deleted before continuing. See [deleteBucket()](https://docs.aws.amazon.com/aws-sdk-php/v3/api/api-s3-2006-03-01.html#deletebucket) for additional details, syntax, and examples.
+
+### Syntax
+
+    $client->deleteBucket([
+        'Bucket' => 'bucket-label'
+    ]);
+
+-  **Bucket (required):** The label of the bucket you wish to delete.
+
+### Example
+
+Delete the bucket labeled "example-bucket":
+
+    $client->deleteBucket(['Bucket' => 'example-bucket']);
+
+## List Objects
+
+Outputs all the objects within a bucket (and with a certain prefix, if specified). See [listObjects()](https://docs.aws.amazon.com/aws-sdk-php/v3/api/api-s3-2006-03-01.html#listobjects) for additional details, syntax, and examples.
+
+### Syntax
+
+    $client->listObjects([
+        'Bucket' => 'bucket-label',
+        'Prefix' => 'object-prefix'
+    ]);
+
+-  **Bucket [required]:** The label of the bucket you wish to use.
+-  **Prefix:** The optional prefix (or pseudo path) of objects you list to view within a bucket. To view all objects, omit the Prefix parameter.
+
+### Examples
+
+-  **List all objects:** List all objects within the bucket called "example-bucket":
+
+        $result = $client->listObjects(['Bucket' => 'example-bucket']);
+        foreach ($result['Contents'] as $object) {
+            echo $object['Key'] . "\n";
+        }
+
+-  **List all objects within a specific "folder":** List all objects stored in the "assets/" folder within the bucket called "example-bucket". Keep in mind that objects aren't actually stored in folders, but the [prefix value](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-prefixes.html) allows them to appear within a structure.
+
+        $result = $client->listObjects(['Bucket' => 'example-bucket','Prefix' => 'assets/']);
+        foreach ($result['Contents'] as $object) {
+            echo $object['Key'] . "\n";
+        }
+
+## Upload a File as an Object
+
+Uploads a file as an object stored within the specified bucket. See [putObject()](https://docs.aws.amazon.com/aws-sdk-php/v3/api/api-s3-2006-03-01.html#putobject) for additional details, syntax, and examples.
+
+### Syntax
+
+    $client->putObject([
+        'Bucket'     => 'bucket-label',
+        'Key'        => 'object-name',
+        'SourceFile' => '/path/to/file.ext'
+    ]);
+
+-  **Bucket [required]:** The label of the bucket you wish to store the file within.
+-  **Key [required]:** The name of the object you wish to create, including any prefix/path.
+-  **SourceFile [required]:** The filename and path of the file to upload.
+
+### Examples
+
+-  Upload the file "file.txt", which is located in the same directory as my python script, to the bucket called "example-bucket". Name this new object "file.txt", the same as the filename.
```

Comment:
"...python script," should be PHP script, and instead of "my' we should use "the" or "your"